src/Pages/ClientsPage/index.js

```diff
@@ -0,0 +1,114 @@
+import React, { useEffect, useState } from 'react';
+import { DataTable, Pagination } from '../../Components/DataTable';
+import Navbar from '../../Components/Navbar';
+import Search from '../../Components/Search';
+import { GetClients } from '../../Services/ClientsService';
+
+function ClientsPage() {
+  const defaultPageSize = 8;
+  const [clients, setClients] = useState([]);
+  const [sortColumn, setSortColumn] = useState({ path: 'no', order: 'asc' });
```

Comment:
@sherviniv no biggie but could you please change this to reflect what the variable names will be once this is merged

should be  `order` and `sort_by`

see open sort PR here:
https://github.com/FoodIsLifeBGP/banana-rails/pull/168/files#r1375314653